libs/components/indicators/src/lib/modules/alert/fixtures/alert-fixtures.module.ts

```diff
@@ -0,0 +1,13 @@
+import { CommonModule } from '@angular/common';
+import { NgModule } from '@angular/core';
+
+import { SkyAlertModule } from '../alert.module';
+
+import { AlertTestComponent } from './alert.component.fixture';
+
+@NgModule({
+  declarations: [AlertTestComponent],
+  imports: [CommonModule, SkyAlertModule],
+  exports: [AlertTestComponent],
+})
+export class AlertFixturesModule {}
```

Comment:
This fixture module is needed to fix a template issue where the test component could not recognize the `<sky-alert>` component tag.

I'm not sure why this was not causing issues before.

Reply:
I believe that is just possibly Angular Language Service yelling. Was that an IDE warning or were you actually getting it when you ran the build commands? I'm not against adding this - but it may have just been an extension being more needy than what is necessary for the unit test classes.